_topic_map.yml

```diff
@@ -620,18 +620,29 @@ Topics:
         File: ruby_on_rails
       - Name: Setting Up a Nexus Mirror
         File: maven_tutorial
-  - Name: Opening a Remote Shell to Containers
-    File: ssh_environment
-  - Name: Templates
-    File: templates
-  - Name: Service Accounts
-    File: service_accounts
   - Name: Builds
-    File: builds
-  - Name: Managing Images
-    File: managing_images
-  - Name: Quotas and Limit Ranges
-    File: compute_resources
+    Dir: builds
+    Topics:
+      - Name: How Builds Work
+        File: index
+      - Name: Basic Build Operations
+        File: basic_build_operations
+      - Name: Build Inputs
+        File: build_inputs
+      - Name: Build Output
+        File: build_output
```

Comment:
So while I don't think that Build Inputs is perfect, I think it's ok for now. Since this is the first step in breaking this apart and reorganizing, I think keeping it simple is a good thing. However, Brice, your comment made me go back and look at all the titles in this section. I think our standard is to title procedural content with verbs or gerunds (don't care which) and then to title conceptual content with nouns. It allows the user to know what's coming. I am really glad that we're talking about this though and figuring it out.